cpp/protocol/openai_api_protocol.cc

```diff
@@ -0,0 +1,237 @@
+#include "openai_api_protocol.h"
+
+picojson::value LoadJsonFromString(const std::string& json_str, std::string& err) {
+  picojson::value json;
+  err = picojson::parse(json, json_str);
+  return json;
+}
```

Comment:
let us simply move those to json_ffi since we won;t have other varianrs that uses this protocol